wg-deployment/charter.md

```diff
@@ -0,0 +1,43 @@
+# WG Deployment Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+WG Deployment is responsible for Kubeflow release and tooling to deploy apps from the catalog (e.g kfctl).
```

Comment:
nit: What is meant by "Kubeflow release"  here; 
maybe just drop it.

Reply:
removed